<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pagina_inicio</class>
 <widget class="QWidget" name="pagina_inicio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1102</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TCN</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="header_corridas" native="true">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>-10</y>
     <width>1951</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 15px;
border-bottom: 2px solid #E3E3E3;</string>
   </property>
   <widget class="QPushButton" name="boton_rutas">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>20</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background: #1061C4;
	color:WHITE;
	border:none;
	border-radius: 8px;
	font-weight: bold;
	font-size:25px;
}

QPushButton:hover {
    background-color: #0D4FAB;    
}

QPushButton:pressed {
    background-color: #0A3F8A;     
}</string>
    </property>
    <property name="text">
     <string>Rutas</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>recursos/recursos_empresa/entrega.png</normaloff>recursos/recursos_empresa/entrega.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_corridas">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>20</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background: #1061C4;
	color:WHITE;
	border:none;
	border-radius: 8px;
	font-weight: bold;
	font-size: 25px;
}

QPushButton:hover {
    background-color: #0D4FAB;    
}

QPushButton:pressed {
    background-color: #0A3F8A;     
}</string>
    </property>
    <property name="text">
     <string>Corridas</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>recursos/recursos_empresa/calendario.png</normaloff>recursos/recursos_empresa/calendario.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_operadores">
    <property name="geometry">
     <rect>
      <x>1390</x>
      <y>20</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background: #1061C4;
	color:WHITE;
	border:none;
	border-radius: 8px;
	font-weight: bold;
	font-size: 25px;
}

QPushButton:hover {
    background-color: #0D4FAB;    
}

QPushButton:pressed {
    background-color: #0A3F8A;     
}</string>
    </property>
    <property name="text">
     <string>Operadores</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>recursos/recursos_empresa/conductor.png</normaloff>recursos/recursos_empresa/conductor.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_autobuses">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>20</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background: #1061C4;
	color:WHITE;
	border:none;
	border-radius: 8px;
	font-weight: bold;
	font-size: 25px;
}

QPushButton:hover {
    background-color: #0D4FAB;    
}

QPushButton:pressed {
    background-color: #0A3F8A;     
}</string>
    </property>
    <property name="text">
     <string>Autobuses</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>recursos/recursos_empresa/autobus.png</normaloff>recursos/recursos_empresa/autobus.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_inicio">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background: #1061C4;
	color:WHITE;
	border:none;
	border-radius: 8px;
	font-weight: bold;
	font-size: 25px;
}

QPushButton:hover {
    background-color: #0D4FAB;    
}

QPushButton:pressed {
    background-color: #0A3F8A;     
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>recursos/recursos_empresa/casa-silueta-negra-sin-puerta.png</normaloff>recursos/recursos_empresa/casa-silueta-negra-sin-puerta.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_reservaciones">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background: #1061C4;
	color:WHITE;
	border:none;
	border-radius: 8px;
	font-weight: bold;
	font-size: 25px;
}

QPushButton:hover {
    background-color: #0D4FAB;    
}

QPushButton:pressed {
    background-color: #0A3F8A;     
}</string>
    </property>
    <property name="text">
     <string>Reservaciones</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>recursos/recursos_empresa/boleto.png</normaloff>recursos/recursos_empresa/boleto.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_logo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>recursos/recursos_empresa/logo.png</pixmap>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_opfiltro_2" native="true">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>130</y>
     <width>1491</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #ffffff;
border: 2px solid #dcdfe6;
border-radius: 15px;
gridline-color: #e0e0e0;
outline: none;
color: #606266;
border-radius: 15px;
border-bottom: 1px solid #E3E3E3;

</string>
   </property>
   <widget class="QLabel" name="label_estatico_titulo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>551</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>59</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none</string>
    </property>
    <property name="text">
     <string>Inicio</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_comprarboleto">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>10</y>
      <width>241</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background: #1061C4;
	color:WHITE;
	border:none;
	border-radius: 8px;
	font-weight: bold;
	font-size: 25px;
}

QPushButton:hover {
    background-color: #0D4FAB;    
}

QPushButton:pressed {
    background-color: #0A3F8A;     
}</string>
    </property>
    <property name="text">
     <string>Comprar boleto</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>recursos/recursos_empresa/boleto.png</normaloff>recursos/recursos_empresa/boleto.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_estatico_operadores">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>650</y>
     <width>318</width>
     <height>68</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>41</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Operadores</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="QtableWidget_operadores">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>730</y>
     <width>681</width>
     <height>291</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableView {
    background: #ffffff;
    border: 2px solid #e6e8ec;
    border-radius: 8px;
    gridline-color: #f1f1f3;
    selection-background-color: #e8f0fe;
    selection-color: #1a1a1a;
    font-size: 18px;
    color: #333;
    alternate-background-color: #fafbff;
    show-grid: false;
}

QTableView::item {
    padding: 8px 12px;
    border: 0px;
}

QTableView::item:hover {
    background: #f5f8ff;
}

QTableView::item:selected {
    background: #e2edff;
    color: #1a1a1a;
}

QHeaderView::section {
    background: #ffffff;
    color: #6b7280;
    padding: 10px 12px;
    border: 0;
    border-bottom: 2px solid #e6e8ec;
    font-size: 20px;
    font-weight: 600;
}

QHeaderView::section:hover {
    background: #f7f9fc;
}

QHeaderView {
    background: #ffffff;
    border: 0;
}

QTableView::indicator {
    width: 18px;
    height: 18px;
}

QHeaderView::section:first {
    border-top-left-radius: 15px;
}

QHeaderView::section:last {
    border-top-right-radius: 15px;
}


QScrollBar:vertical {
    background: transparent;
    width: 10px;
    margin: 0px;
}

QScrollBar::handle:vertical {
    background: #cfd3da;
    min-height: 25px;
    border-radius: 5px;
}

QScrollBar::handle:vertical:hover {
    background: #b5bac4;
}

QScrollBar::add-line,
QScrollBar::sub-line {
    background: none;
    height: 0px;
}
</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>338</number>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>N째 Operador</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nombre</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="QtableWidget_corridasact">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>340</y>
     <width>1111</width>
     <height>291</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableView {
    background: #ffffff;
    border: 2px solid #e6e8ec;
    border-radius: 8px;
    gridline-color: #f1f1f3;
    selection-background-color: #e8f0fe;
    selection-color: #1a1a1a;
    font-size: 18px;
    color: #333;
    alternate-background-color: #fafbff;
    show-grid: false;
}

QTableView::item {
    padding: 8px 12px;
    border: 0px;
}

QTableView::item:hover {
    background: #f5f8ff;
}

QTableView::item:selected {
    background: #e2edff;
    color: #1a1a1a;
}

QHeaderView::section {
    background: #ffffff;
    color: #6b7280;
    padding: 10px 12px;
    border: 0;
    border-bottom: 2px solid #e6e8ec;
    font-size: 20px;
    font-weight: 600;
}

QHeaderView::section:hover {
    background: #f7f9fc;
}

QHeaderView {
    background: #ffffff;
    border: 0;
}

QTableView::indicator {
    width: 18px;
    height: 18px;
}

QHeaderView::section:first {
    border-top-left-radius: 15px;
}

QHeaderView::section:last {
    border-top-right-radius: 15px;
}


QScrollBar:vertical {
    background: transparent;
    width: 10px;
    margin: 0px;
}

QScrollBar::handle:vertical {
    background: #cfd3da;
    min-height: 25px;
    border-radius: 5px;
}

QScrollBar::handle:vertical:hover {
    background: #b5bac4;
}

QScrollBar::add-line,
QScrollBar::sub-line {
    background: none;
    height: 0px;
}
</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>221</number>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>N째 Corrida</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Fecha y Hora Salida</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Origen</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Destino</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Autobus</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_estatico_corridas">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>260</y>
     <width>448</width>
     <height>68</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>41</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Corridas activas</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_bfecha">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>280</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
QComboBox {
    background-color: #ffffff;
    border: 2px solid #dcdfe6;
    border-radius: 6px;
    padding: 6px 12px;
    font-size: 18px;
    color: #606266;
    min-height: 18px;
}

QComboBox:hover {
    border-color: #c0c4cc;
}

QComboBox:focus {
    border-color: #409eff;
}

QComboBox::drop-down {
    border: none;
    width: 25px;
}

QComboBox::down-arrow {
    image: none;
    border-left: 4px solid transparent;
    border-right: 4px solid transparent;
    border-top: 4px solid #909399;
}

QComboBox QAbstractItemView {
    border: 1px solid #dcdfe6;
    border-radius: 6px;
    background-color: white;
    outline: none;
    selection-background-color: #409eff;
    selection-color: white;
}
</string>
   </property>
   <property name="placeholderText">
    <string>Buscar por fecha</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="QtableWidget_pasajeros">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>730</y>
     <width>1111</width>
     <height>291</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableView {
    background: #ffffff;
    border: 2px solid #e6e8ec;
    border-radius: 8px;
    gridline-color: #f1f1f3;
    selection-background-color: #e8f0fe;
    selection-color: #1a1a1a;
    font-size: 18px;
    color: #333;
    alternate-background-color: #fafbff;
    show-grid: false;
}

QTableView::item {
    padding: 8px 12px;
    border: 0px;
}

QTableView::item:hover {
    background: #f5f8ff;
}

QTableView::item:selected {
    background: #e2edff;
    color: #1a1a1a;
}

QHeaderView::section {
    background: #ffffff;
    color: #6b7280;
    padding: 10px 12px;
    border: 0;
    border-bottom: 2px solid #e6e8ec;
    font-size: 20px;
    font-weight: 600;
}

QHeaderView::section:hover {
    background: #f7f9fc;
}

QHeaderView {
    background: #ffffff;
    border: 0;
}

QTableView::indicator {
    width: 18px;
    height: 18px;
}

QHeaderView::section:first {
    border-top-left-radius: 15px;
}

QHeaderView::section:last {
    border-top-right-radius: 15px;
}


QScrollBar:vertical {
    background: transparent;
    width: 10px;
    margin: 0px;
}

QScrollBar::handle:vertical {
    background: #cfd3da;
    min-height: 25px;
    border-radius: 5px;
}

QScrollBar::handle:vertical:hover {
    background: #b5bac4;
}

QScrollBar::add-line,
QScrollBar::sub-line {
    background: none;
    height: 0px;
}
</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>276</number>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>N째 boleto</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>N째 Asiento</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nombre </string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Edad</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_estatico_pasajeros">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>650</y>
     <width>269</width>
     <height>68</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>41</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Pasajeros</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_boletospasajeros">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>650</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background: #1061C4;
	color:WHITE;
	border:none;
	border-radius: 8px;
	font-weight: bold;
	font-size: 25px;
}

QPushButton:hover {
    background-color: #0D4FAB;    
}

QPushButton:pressed {
    background-color: #0A3F8A;     
}</string>
   </property>
   <property name="text">
    <string>Boletos</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>recursos/recursos_empresa/boleto.png</normaloff>recursos/recursos_empresa/boleto.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_corridasoperadores">
   <property name="geometry">
    <rect>
     <x>1630</x>
     <y>650</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background: #1061C4;
	color:WHITE;
	border:none;
	border-radius: 8px;
	font-weight: bold;
	font-size: 25px;
}

QPushButton:hover {
    background-color: #0D4FAB;    
}

QPushButton:pressed {
    background-color: #0A3F8A;     
}</string>
   </property>
   <property name="text">
    <string>Corridas</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>recursos/recursos_empresa/boleto.png</normaloff>recursos/recursos_empresa/boleto.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>1280</x>
     <y>335</y>
     <width>506</width>
     <height>326</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../recursos/recursos_usuario/image(1).png</pixmap>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>header_corridas</zorder>
  <zorder>widget_opfiltro_2</zorder>
  <zorder>label_estatico_operadores</zorder>
  <zorder>QtableWidget_operadores</zorder>
  <zorder>QtableWidget_corridasact</zorder>
  <zorder>label_estatico_corridas</zorder>
  <zorder>comboBox_bfecha</zorder>
  <zorder>QtableWidget_pasajeros</zorder>
  <zorder>label_estatico_pasajeros</zorder>
  <zorder>boton_boletospasajeros</zorder>
  <zorder>boton_corridasoperadores</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
